motionlayout/src/main/java/com/google/androidstudio/motionlayoutexample/utils/DrawerContent.kt

```diff
@@ -17,14 +17,14 @@
 package com.google.androidstudio.motionlayoutexample.utils
 
 import android.content.Context
-import android.support.constraint.motion.MotionLayout
-import android.support.v4.widget.DrawerLayout
+import androidx.drawerlayout.widget.DrawerLayout
 import android.util.AttributeSet
 import android.view.View
+import androidx.constraintlayout.motion.widget.MotionLayout
 
 class DrawerContent @JvmOverloads constructor(
         context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0
-) : MotionLayout(context, attrs, defStyleAttr), DrawerLayout.DrawerListener {
+) : MotionLayout(context, attrs, defStyleAttr), androidx.drawerlayout.widget.DrawerLayout.DrawerListener {
```

Comment:
Please import androidx.drawerlayout.widget.DrawerLayout

Reply:
It is imported